### ISSUE TITLE
[FxA-Settings] Missing tab functionality in Accessibility

### ISSUE THREAD

**Issue by rbillings** [mozilla/fxa#7489]
## Description

DevTools > Accessibility > Show Tabbing Order before.

Looks like several of the elements aren’t tabbable (ie: “display name”, “password”). You can see from this tiny screenshot that we go from “Bento” > Profile pic > Secondary email >> Make Primary | Delete icon.

Submitted by: pdehaan

### Steps to reproduce

Clear steps describing how to reproduce the issue. Providing a URL, screenshot.

### Expected result

All element can be tabbed through

### Actual result

Missing display name and password

<img width="329" alt="Screen Shot 2021-02-09 at 12 40 56 PM" src="https://user-images.githubusercontent.com/737878/107425786-42b98e80-6ad4-11eb-8c5a-d59fe599b571.png">

### Environment

macOS

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-3027)
┆Issue Number: FXA-3027

**Comment by vasilicatamas** [comment]
This issue is no longer reproducible on Windows 10 x64 and macOS 10.14.6. All the Settings page elements can be tabbed through.

![Screenshot_34](https://user-images.githubusercontent.com/18114210/107538607-d40c2c00-6bcc-11eb-8d6f-e7c8415a56eb.png)
